src/main/java/org/jenkinsci/test/acceptance/po/Login.java

```diff
@@ -40,7 +44,10 @@ public Login doLogin(String user, String password){
     public Login doLoginDespiteNoPaths(String user, String password){
         driver.findElement(by.name("j_username")).sendKeys(user);
         driver.findElement(by.name("j_password")).sendKeys(password);
-        driver.findElement(by.name("Submit")).click();
+        WebElement we = driver.findElement(by.name("Submit"));
+        // for some reason submit() is bogus
+        we.click();
```

Comment:
Is there a reason this is not using the `clickAndWaitToBecomeStale()`?

Reply:
because this is using the `WebElement` directly and not a control.